src/io/parser.cpp

```diff
@@ -235,7 +235,7 @@ Parser* Parser::CreateParser(const char* filename, bool header, int num_features
   int num_col = 0;
   DataType type = GetDataType(filename, header, lines, &num_col);
   if (type == DataType::INVALID) {
-    Log::Fatal("Unknown format of training data.");
+    Log::Fatal("Unknown format of training data. Only CSV, TSV, and LibSVM formats are supported.");
```

Comment:
Please update wording here to something like
```
LibSVM (zero-based) / TSV / CSV formatted raw text file
```
![image](https://user-images.githubusercontent.com/25141164/130357392-230c83b7-5de4-4fdc-90db-deff6dfb9cbd.png)

https://github.com/microsoft/LightGBM/blob/master/docs/Python-Intro.rst#data-interface

Reply:
what is "TXT format"? It seems like that  line is confusing the difference between file format and the extension in the file's name.

Reply:
Yeah, I agree, that's why I asked to update it omitting file extension.

Reply:
OH I misunderstood. I thought by "here" you meant "update this line in the diff to match that line in the docs", but I see now that you meant the opposite. My mistake, I'll update the docs accordingly, thanks!

Reply:
Updated in https://github.com/microsoft/LightGBM/pull/4545/commits/33448b6d2a396447e6515b8357d69fa1cfc7c351. If I still misunderstood what you're requesting, it's ok with me if you push directly to this branch.

Reply:
I'm sorry for initial confusing suggestion! Thanks, you've made exactly what I wanted to change.